torch/distributions/exponential.py

```diff
@@ -39,3 +40,11 @@ def log_prob(self, value):
 
     def entropy(self):
         return 1.0 - torch.log(self.rate)
+
+    @lazy_property
+    def natural_params(self):
+        V1 = -self.rate
+        return (V1, )
```

Comment:
Can you simplify those methods to one liners like `(-self.rate,)`? Also, do we really want them to be lazy properties? They will hold on to their results, consuming more memory than necessary.